### DISCUSSION TITLE
Cruiser and bicycle tour - compared to self created Komoot tour

### DISCUSSION THREAD

**Discussion by WolfgangZ42** [devemux86/cruiser#206] [Q&A] [answered]
I never thought of organizing my bicycle tours with Cruiser - have been using Komoot for a long time for that. Yesterday I gave Cruiser a chance when planning a tour. Set some BRouter options for bicycles for that and compared the result (magenta) with the created (several waypoints!) Komoot route (dashed blue): 
<img width="200" src="https://github.com/devemux86/cruiser/assets/121806238/19846cb2-4469-4cc9-83df-e311d17beaa1"/>
It was of course not really a match - but some parts of the Komoot route I have subsequently adapted to the BRouter course.

I will not switch from Komoot to Cruiser now - but respect BRouter's performance here.

**Comment by devemux86** [comment, marked as answer]
@WolfgangZ42 thanks for the tests!

You can also try the @poutnikl [BikeProfiles](https://github.com/poutnikl/Brouter-profiles/tree/master/BikeProfiles) to see how they compare to the defaults.
(via "Open external profile" function)

Also have you tried the [CycleStreets](https://www.cyclestreets.net/) routes?
